### PR TITLE
[Doc] Improve description of the pos parameter in gui

### PR DESCRIPTION
**Pull request by houkensjtu**
The current description of parameter pos in gui.text doesn't explicitly show it's intended value range. 
Same issue exists for other objects. I can make a through chk and update if you think the change is valuable.
Thank you!

Related issue = #

[[Click here for the format server]](http://kun.csail.mit.edu:31415/)

----

### CODE REVIEW

File: docs/gui.rst [hunk 1]
```diff
@@ -202,7 +202,7 @@ Paint on a window
 
     :parameter gui: (GUI) the window object
     :parameter content: (str) the text to draw
-    :parameter pos: (tuple of 2) the top-left point position of the fonts / texts
+    :parameter pos: (tuple of 2, range [0,1] ) the relative position of the top-left point of the fonts / texts
```
Comment (archibate) [inline docs/gui.rst:205]:
```suggestion
    :parameter pos: (tuple of 2, range [0,1]) the relative position of the top-left point of the fonts / texts
```

**Review by archibate — COMMENTED** [review]
Good idea! LGTM but I've several concerns:

1. There're more than one place we have `pos` in GUI doc, will we modify them too?
1. Thank for clarifying `[0, 1]`, shall we also claim that X is from left to right and Y is from down to up?

So I'd suggest - instead of repeating ourselves by adding `range [0,1]` to all `pos` occasions, how about to **add a brief introduction on what's the coordinate system Taichi GUI uses** at the beginning of this doc, so that users will automatically know what the following `pos` means?

File: docs/gui.rst (hunk 1 above)
Comment (houkensjtu) [reply docs/gui.rst]: I agree! I submitted a new commit with a brief introduction section. The explanation of pos goes into a Note section. Please review.

File: docs/gui.rst [hunk 2]
```diff
@@ -63,6 +63,14 @@ Create a window
 Paint on a window
 -----------------
 
+Paint simple geometric objects on a window. Taichi has internal support for lines, triangles, rectangles, circles and texts.
+
+
+.. note::
+
+    The position parameter ``pos`` expects an input of a tuple of 2, whose values are the relative position of the object. 
+    ``(0.0, 0.0)`` stands for the lower left corner of the window, and ``(1.0, 1.0)`` stands for the upper right corner.
```
Comment (archibate) [inline docs/gui.rst:72]: May also mention in `gui.get_cursor_pos()` as it follows this law too?

File: docs/gui.rst [hunk 3]
```diff
@@ -63,6 +63,14 @@ Create a window
 Paint on a window
 -----------------
 
+Paint simple geometric objects on a window. Taichi has internal support for lines, triangles, rectangles, circles and texts.
```
Comment (archibate) [inline docs/gui.rst:66]:
```suggestion
Taichi has internal support for lines, triangles, rectangles, circles and texts. You can easily paint these simple geometries on the window by invoking the following APIs.
```

**Review by archibate — APPROVED** [review]
LGTMig + 2 nits :)

File: docs/gui.rst [hunk 4]
```diff
@@ -63,6 +63,14 @@ Create a window
...
+
+
+.. note::
+
+    The position parameter ``pos`` expects an input of a tuple of 2, whose values are the relative position of the object.
```
Comment (yuanming-hu) [inline docs/gui.rst:71]:
```suggestion
    The position parameter ``pos`` expects an input of a 2-element tuple, whose values are the relative position of the object. 
```

File: docs/gui.rst (hunk 3 above)
Comment (yuanming-hu) [inline docs/gui.rst:66]:
```suggestion
Taichi's GUI supports painting simple geometric objects, such as lines, triangles, rectangles, circles, and text.
```

**Review by yuanming-hu — APPROVED** [review]
Thanks for the PR, and welcome on board!

This mostly LGTM. I added some minor wording fixes. This PR can be merged if the suggestions are committed.

**Pull request merged**